contents/docs/getting-started/estimating-usage-costs.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Estimating Usage & Costs
+nextPage: ./next-steps.mdx
+featuredImage: ./images/docs-groups.png
```

Comment:
The image looks kind of weird since the one above is exactly the same. Maybe let's use the image from the Install step?
<img width="1092" alt="Screenshot 2023-06-30 at 7 57 00 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/ef96175f-0ec8-41b9-a877-e0867498965d">

Reply:
[According to Cory](https://github.com/PostHog/posthog.com/issues/6201#issuecomment-1614004464) this won't be an issue in a couple days.